stl/inc/yvals_core.h

```diff
@@ -351,6 +351,21 @@
 // * unique_copy
 
 #include <vcruntime.h>
+
+// TRANSITION, <vcruntime.h> should define _HAS_CXX23
+#if _HAS_CXX20 && !defined(_HAS_CXX23) \
+    && (defined(_MSVC_LANG) && _MSVC_LANG > 202002L || defined(__cplusplus) && __cplusplus > 202002L)
+#define _HAS_CXX23 1
+#endif
+
+#ifndef _HAS_CXX23
+#define _HAS_CXX23 0
+#endif

```

Comment:
I think it would be a bit easier to read if we would pull the `#ifndef _HAS_CXX23` out
```suggestion
#ifndef _HAS_CXX23
#if _HAS_CXX20 \
    && (defined(_MSVC_LANG) && _MSVC_LANG > 202002L || defined(__cplusplus) && __cplusplus > 202002L)
#define _HAS_CXX23 1
#else
#define _HAS_CXX23 0
#endif
```